lib/ModelMixins/Cesium3dTilesMixin.ts

```diff
@@ -214,7 +212,7 @@ export default function Cesium3dTilesMixin<
       }
 
       if (this.tileset.destroyed) {
-        this.loadTileset();
+        this.loadMapItems(true);
```

Comment:
This is here so we don't get errors thrown in `get mapItems()`.

Not really a big fan of loading things in `get mapItems()` though 😢

Reply:
Why is this necessary? When does a tileset get `destroyed`?